### DISCUSSION TITLE
Different tables at the same event

### DISCUSSION THREAD

**Discussion by ghost** [Playabl-io/playabl#227] [Ideas]
Hi, 
 we are looking at Playbl as an alternative to meetup.com for our face to face RPG community. 
We organize a monthly public game day where we play four games in parallel. Think of it like a really tiny con. On meetup.com we need to keep track of players signing up to each game by modifying the event description, because there's no feature to support something like this. We just get a signup for the whole event, then each player tells us what they want to play.

Is there any plan for such a feature here on Playbl? Like, breakout sessions, con organization, something on this line? 
Thanks!

**Comment by jongrim** [comment]
If I'm understanding correctly, this should work as-is with Playabl. You can set up the four games each as their own game, and create the session for that date. Each game will have it's own sign-up list so members can RSVP directly to the game they are interested in. Playabl was built with online playing in mind first, but you could easily write "in person" or "table 1" in the field for virtual tabletop. Each month you can either make a new game, or you can add sessions to the existing game.

Does that meet your needs? Is it clear? [I have a small guide written on creating games](https://docs.playabl.io/guides/games/creating-games.html), but I admit the documentation is not extremely thorough, so let me know if you have questions. You can also cancel a game after it's made so there's no real harm in experimenting and trying things out.

[One other thing I'll mention is you might want to also check out the Events feature](https://docs.playabl.io/guides/events/running-events.html). Events let you schedule multiple games and sessions and set a common access time for each. I could see this being handy if you want to try and control opening access to members so that it happens for all four games at the same time. In the normal game creation flow, the RSVP available time is calculated immediately when created and it's not currently an option to set a specific date and time. That's something that could possibly be added in the future, so let me know if that is of interest.

**Comment by ghost** [comment]
Wonderful, we'll give it a shot, thanks!
